### ISSUE TITLE
Provisioning broken can't read /vagrant/shell/*

### ISSUE THREAD

**Issue by Amparose** [puphpet/puphpet#1740]
`vagrant up` fails at provisioning step with message:

```
==> default: stdin: is not a tty
==> default: sed: can't read /vagrant/shell/*: No such file or directory
```

I haven't changed anything after downloading the archive from the puphpet website. It seems to mount the shared folders in a way that "puphpet" is missing from the file path in the error above so actual path is (I think):

```
/vagrant/puphpet/shell/*
```

Here is the config.yaml:

https://gist.github.com/Amparose/2c1587fff00100aa1d5f

I have confirmed this on two separate machines, OSX and Windows. Any help would be appreciated as I'm a bit lost.

**Comment by Jesipov** [comment]
I have the same problem #1739
And that caused it #1738

I temporarily removed this line so I could work again.

**Comment by c0ntax** [comment]
I just came here hunting for a solution to the same problem...

**Comment by Amparose** [comment]
Yeah I went hunting and suspected that line 216 in `vagrant/Vagrantfile-local` was the issue. What does it actually do?

**Comment by emmio-micke** [comment]
I have the same problem, my config:

https://gist.github.com/emmio-micke/5f6558c4121034e4d1a6

I guess it's due to the work with windows described here:

http://blog.puphpet.com/blog/2015/06/25/windows-symlinks/

**Comment by jtreminio** [comment]
Fixed by 1405530

Sorry all. That's what happens when coding at 3am.

**Comment by jtreminio** [comment]
@Amparose it removes all instances of `\r` which is the Windows newline character.

`\n` is unaffected.

**Comment by YanshuoH** [comment]
Hello, it seems that there's always a little problem after the fix if I haven't done anything stupid:
```
==> default: couldn't edit /vagrant/puphpet/shell/ascii-art: not a regular file
```

**Comment by jtreminio** [comment]
Fixed in f22c8b6
